prow/e2e-simpleTests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Copyright 2017 Istio Authors
```

Comment:
2018.

Offhand it looks like prow/config.yaml in the test-infra repo might need to be updated to reflect the rename of e2e-smoke.sh.

Reply:
Yup. https://github.com/istio/test-infra/pull/694/